pkg/resources/counts/buffer.go

```diff
@@ -6,35 +6,61 @@ import (
 	"github.com/rancher/apiserver/pkg/types"
 )
 
-func buffer(c chan types.APIEvent) chan types.APIEvent {
+// debounceDuration determines how long events will be held before they are sent to the consumer
+var debounceDuration = 5 * time.Second
+
+// countsBuffer creates an APIEvent channel with a buffered response time (i.e. replies are only sent once every second)
+func countsBuffer(c chan Count) chan types.APIEvent {
 	result := make(chan types.APIEvent)
 	go func() {
 		defer close(result)
-		debounce(result, c)
+		debounceCounts(result, c)
 	}()
 	return result
 }
 
-func debounce(result, input chan types.APIEvent) {
-	t := time.NewTicker(time.Second)
+// debounceCounts converts counts from an input channel into an APIEvent, and updates the result channel at a reduced pace
+func debounceCounts(result chan types.APIEvent, input chan Count) {
+	// counts aren't a critical value. To avoid excess UI processing, only send updates every 5 seconds
```

Comment:
Nit: 
In case debounceDuration is changed in the future.
```suggestion
	// counts aren't a critical value. To avoid excess UI processing, only send updates after debounceDuration has elapsed.

Reply:
Done - good call.